src/patients/patient-slice.ts

```diff
@@ -71,7 +71,11 @@ export const createPatient = (patient: Patient, history: any): AppThunk => async
   )
 }
 
-export const updatePatient = (patient: Patient, history: any): AppThunk => async (dispatch) => {
+export const updatePatient = (
```

Comment:
I think instead of having a flag for for `relatedPersons` we should dispatch a different action. 

I think creating a function like the below gives us more flexibility and better traceability into what actions have been executed..
```ts
export const addRelatedPerson = (patient: Patient, history: any): AppThunk = async (dispatch) => {
 ...
}
```

We can then create a helper function to do any common actions between the update code and the add related persons code.

Reply:
added